libtiledbvcf/src/dataset/tiledbvcfdataset.cc

```diff
@@ -1059,8 +1080,8 @@ std::unordered_map<uint32_t, SafeBCFHdr> TileDBVCFDataset::fetch_vcf_headers_v4(
       all_samples,
       first_sample,
       utils::memory_usage_str());
-  // Grab a read lock of concurrency so we don't destroy the vcf_header_array
-  // during fetching
+  // Grab a read lock of concurrency so we don't destroy the
```

Comment:
A lot of changes in this file are line wrapping only. Please keep the same line wrapping to reduce noise in the code changes.